analysis/rogueoutlaw/src/modules/features/RollTheBonesCastTracker.js

```diff
@@ -9,10 +9,24 @@ import { ROLL_THE_BONES_BUFFS, ROLL_THE_BONES_DURATION } from '../../constants';
 
 export const ROLL_THE_BONES_CATEGORIES = {
   LOW_VALUE: 'low',
-  MID_VALUE: 'mid',
   HIGH_VALUE: 'high',
 };
 
+let BROADSIDE_VALUE = { sleight_of_hand: 2, base: 3 };
+let TRUE_BEARING_VALUE = { sleight_of_hand: 2, base: 3 };
+let RUTHLESS_PRECISION_VALUE = { sleight_of_hand: 2, base: 2 };
+let SKULL_AND_CROSSBONES_VALUE = { sleight_of_hand: 2, base: 2 };
+let BURIED_TREASURE_VALUE = { sleight_of_hand: 1, base: 1 };
+let GRAND_MELEE_VALUE = { sleight_of_hand: 1, base: 1 };
+
+var BUFF_VALUE_BY_ID = {};
+BUFF_VALUE_BY_ID[SPELLS.BROADSIDE.id] = BROADSIDE_VALUE;
+BUFF_VALUE_BY_ID[SPELLS.TRUE_BEARING.id] = TRUE_BEARING_VALUE;
+BUFF_VALUE_BY_ID[SPELLS.RUTHLESS_PRECISION.id] = RUTHLESS_PRECISION_VALUE;
+BUFF_VALUE_BY_ID[SPELLS.SKULL_AND_CROSSBONES.id] = SKULL_AND_CROSSBONES_VALUE;
+BUFF_VALUE_BY_ID[SPELLS.BURIED_TREASURE.id] = BURIED_TREASURE_VALUE;
+BUFF_VALUE_BY_ID[SPELLS.GRAND_MELEE.id] = GRAND_MELEE_VALUE;
+
```

Comment:
use `const` instead of `let` if you're not going to be reassigning the value. `var` should not be used in general, use `let` or `const` instead.

`var` has global or function scope, not lexical scope (as used by most languages by default), which can lead to very surprising behaviors.

in this particular case, it looks like these could all be `const`, but i'm not going to nitpick as long as the use of `var` is changed.

Reply:
TIL, thanks for the rundown :)